rust/src/router/consumer.rs

```diff
@@ -442,6 +444,7 @@ impl ConsumerStats {
         match self {
             ConsumerStats::JustConsumer((consumer_stat,)) => consumer_stat,
             ConsumerStats::WithProducer((consumer_stat, _)) => consumer_stat,
+            ConsumerStats::MultipleConsumers(consumer_stats) => &consumer_stats[0],
```

Comment:
`consumer_stats()` returns a single consumer stat by definition, hence here it's returning the first one. All consumer stats are accessible from the `MultipleConsumers` variant directly.

Reply:
Is `ConsumerStats::WithProducer` used anywhere now? Two entries meant consumer+producer before :thinking:

Reply:
Yes, `SimpleConsumer` and `SimulcastConsumer` returns the pair of recv-send streams stats.

Reply:
But with this PR there is nothing that ever constructs `ConsumerStats::WithProducer`. Should you check the kind of consumer and then produce either `ConsumerStats::WithProducer` or `ConsumerStats::MultipleConsumers` maybe?

Reply:
`MultipleConsumer` must remain. Yes, I'll create one or another based on the consumer type.

Reply:
Done